pyproject.toml

```
@@ -5,3 +5,50 @@ python_files = [
     "tests.py",
     "test_*.py",
 ]
+
+# ==== black ====
+[tool.black]
+line-length = 119
+target-version = ['py311']
+
+
+# ==== isort ====
+[tool.isort]
+profile = "black"
+line_length = 119
+known_first_party = [
+    "commcare_connect",
+    "config",
+]
+skip = ["venv/"]
+skip_glob = ["**/migrations/*.py"]
+
+# ==== PyLint ====
+[tool.pylint.MASTER]
+load-plugins = [
+    "pylint_django",
+    "pylint_celery",
+]
+django-settings-module = "config.settings.local"
```

This also requires `__init__.py` file in `connectid` folder to make it a python module.

```suggestion
django-settings-module = "connectid.settings"
```